src/vs/workbench/contrib/welcome/telemetryOptOut/browser/telemetryOptOut.ts

```diff
@@ -136,9 +140,10 @@ export abstract class AbstractTelemetryOptOut implements IWorkbenchContribution
 					},
 					{
 						label: noLabel,
-						run: () => {
+						run: async () => {
 							logTelemetry(true);
 							this.configurationService.updateValue('telemetry.enableTelemetry', false);
+							await this.jsonEditingService.write(this.environmentService.argvResource, [{ path: ['enable-crash-reporter'], value: false }], true);
 							this.configurationService.updateValue('telemetry.enableCrashReporter', false);
```

Comment:
@deepak1556 the entire `telemetry.enableCrashReporter` config should go away, no? Otherwise a user may assume that changing `telemetry.enableCrashReporter` disables the crash reporter?